_includes/how-to-guides/databases/postgis-installation.md

```diff
@@ -39,7 +39,7 @@ Once your application has been deployed, you can find your PostGIS file in `/etc
 
 ## Manual installation
```

Comment:
please comment out this whole `manual` section for installing on 16.04

Reply:
Ok